src/test/java/it/gov/pagopa/cgn/portal/scheduler/SuspendDiscountsWithoutAvailableBucketCodesJobTest.java

```diff
@@ -45,7 +45,7 @@ void Execute_ExecuteJob_DoNotSuspendDiscountIfGivenDaysNotPassed() throws IOExce
             s.setExpiredAt(OffsetDateTime.now());
             discountBucketCodeSummaryRepository.save(s);
         });
-        testJob(DiscountStateEnum.PUBLISHED);
+        testJob(DiscountStateEnum.SUSPENDED);
```

Comment:
We should keep code clean by renaming this test if now the discount has been suspended.
I suggest from this `Execute_ExecuteJob_DoNotSuspendDiscountIfGivenDaysNotPassed` to something like this `Execute_ExecuteJob_AlwaysSuspendDiscountIfCodesAreFinished`